test/fixtures/any_char.ftl

```diff
@@ -0,0 +1,8 @@
+# ERROR       ↓ BEL, U+0007
```

Comment:
Adjust the comments such that these aren't errors anymore? All of them in this file.

Reply:
Oops, thanks :)